cogs/economy.py

```diff
@@ -508,8 +508,11 @@ def already_in_list(self, accounts, user):
     async def registercredits(self, ctx, credits : int):
         """Credits given on registering an account"""
         server = ctx.message.server
-        self.settings[server.id]["REGISTER_CREDITS"] = credits
-        await self.bot.say("Registering an account will now give " + str(credits) + " credits.")
+        if credits >= 0:
+            self.settings[server.id]["REGISTER_CREDITS"] = credits
+            await self.bot.say("Registering an account will now give " + str(credits) + " credits.")
```

Comment:
If you don't do that, setting credits lower than 0 will save changes without the bot saying anything